includes/Admin/Settings_Screens/Product_Sync.php

```diff
@@ -270,6 +270,22 @@ public function get_settings() {
 				'default' => 'yes',
 			],
 
+			[
+				'id'       => \WC_Facebookcommerce_Integration::SETTING_FB_RETAILER_ID_TYPE,
+				'title'    => __( 'Product Identifier', 'facebook-for-woocommerce' ),
+				'type'     => 'select',
+				'class'    => 'product-sync-field',
+				'desc_tip' => __( 'WooCommerce product attribute mapped to retailer_id on Facebook.
+					WARNING: use with caution, as changing this option after uploading products to Facebook may
```

Comment:
Perhaps it would be good to move the Warning to a normal description (`desc`) to make it more easy for merchants to see the risks.